src/GraphQL/Execution/DocumentExecuter.cs

```diff
@@ -373,6 +373,18 @@ public Variables GetVariableValues(ISchema schema, Variables variables, Inputs i
         public object GetVariableValue(ISchema schema, Variable variable, object input)
         {
             var type = schema.FindType(variable.Type.FullName);
+
+            if (type == null && !variable.Type.AllowsNull)
+            {
+                var nonNullType = new VariableType
+                {
+                    Name = variable.Type.Name,
+                    IsList = variable.Type.IsList,
+                    AllowsNull = true,
+                };
+                type = schema.FindType(nonNullType.FullName);
+            }
```

Comment:
Do you have an example of this failing and where it is needed?  This seems like an odd and incorrect change.  "If the type can't be found and its non-null, create a nullable variable type".  Is the issue the name is coming through as `MyType!` and the lookup only knows about `MyType`?